sound/soc/sdca/sdca_device.c

```diff
@@ -44,7 +44,7 @@ bool sdca_device_quirk_match(struct sdw_slave *slave, enum sdca_quirk quirk)
 		if (id->part_id != 0x712 &&
 		    id->part_id != 0x713 &&
 		    id->part_id != 0x716 &&
-		    id->part_id == 0x717)
+		    id->part_id != 0x717)
```

Comment:
I believe the change belongs to `ASoC: SDCA: add quirk function for RT712_VB match`